bundles/org.openhab.ui/web/src/pages/settings/rules/rule-module-popup.vue

```diff
@@ -18,7 +18,7 @@
         </f7-nav-right>
       </f7-navbar>
       <f7-block v-if="ruleModule" class="no-margin no-padding">
-        <f7-col v-if="!isSceneModule && currentRuleModuleType" class="margin-top">
+        <f7-col v-if="!currentRuleModuleType" class="margin-top">
```

Comment:
@florian-h05 this reversed the logic, and made the inputs to show up when they shouldn't, and not show up when they should.

Reply:
Thx foir the catch, fixed by #2505.